### PR TITLE
Minimap and info

### LINKED ISSUE #19: minimap & info table
## Minimap

Enables users to view which section of the graph they are currently in. The utility of this feature scales with the size of the graph and the number of events. 
  
Goals:  

* Show where the user is in relation to the graph
* Allow user to navigate through the graph using the Minimap    
  
    
## Info Table

A small table on the top right corner of the graph to show which event is selected by the user. Currently, the table will show the id of the event, but more information can be added to it based on user preferences.

### PR DESCRIPTION
**Pull request by SyncInProgress**
A basic design for the Minimap and Information Box (Tooltip)

Goals:
- [x] Minimap shows where the user is in relation to the graph
- [x] Minimap allows the user to navigate through the graph
- [x] Information table to show event information

Closes #19 

Minimap
Tooltip Minimap

### CODE REVIEW

File: frontend/src/components/Graph.tsx [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
+  const { item } = context
+  const model = item && item.getModel()
+
+  return (
+    <div>
+      <Card id={model.id} />
+    </div>
+  )
+}
+
```
Comment (IdreesSamadi) [inline frontend/src/components/Graph.tsx:26]: its better to keep each file slim, and to have one component per file. move the tooltip to its own file and import in graph.tsx.

Comment (IdreesSamadi) [reply frontend/src/components/Graph.tsx:26]: maybe also move the minimap to the right. we have the navigation to the left seem like too many stuff

**Review by IdreesSamadi — CHANGES_REQUESTED** [review]

File: frontend/src/components/Graph.tsx (hunk 1 above)
Comment (RuthgerD) [reply frontend/src/components/Graph.tsx:26]: I vote for bottom right too.

Comment (SyncInProgress) [reply frontend/src/components/Graph.tsx]: image

Comment (IdreesSamadi) [reply frontend/src/components/Graph.tsx]: nice

**Review by IdreesSamadi — APPROVED** [review]

**Pull request merged**